.travis.yml

```diff
@@ -19,3 +19,11 @@ matrix:
     - rvm: jruby-head
     - rvm: rbx-3
   fast_finish: true
+script:
+  # Unit test
+  - bundle exec rake
+  # Install test
+  - gem build rb-inotify.gemspec
+  - gem install rb-inotify-*.gem
```

Comment:
This install test `gem install rb-inotify-*.gem` is to prove the `rb-inotify.gemspec` file is valid, and it can be installed to supported platforms.